project/src/com/company/entities/Savings.java

```diff
@@ -0,0 +1,43 @@
+package com.company.entities;
+
+public class Savings {
+    Account account;
+    double balance;
+
+    //constructor
+    public Savings(Account account) {
+        this.account = account;
+        this.balance = 0;
+    }
+
+    //getters
+    public Account getAccount() {
+        return account;
+    }
+
+    public double getBalance() {
+        return balance;
+    }
+
+    //setters
+    public void setAccount(Account account) {
+        this.account = account;
+    }
+
+    public void setBalance(double balance) {
+        this.balance = balance;
+    }
+
+    //functions
+    public void addSavings(double amount){
```

Comment:
ususally business logic, therefore methods not related to setters, getters are kept in services - SavingService for example

in general, each main entity has its own service where the business logic is kept, you can consider refactor it and have multiple services, then call methods from services in Main

Reply:
I have implemented this way because I use abstract classes and I have some functions inside the classes in order to override them in the derived classes. This is not the case for Savings, but here add and remove savings seemed really close to setters (since they only add or subtract).